markdown/extensions/toc.py

```diff
@@ -297,12 +300,13 @@ def __init__(self, **kwargs):
                         "Function to generate anchors based on header text - "
                         "Defaults to the headerid ext's slugify function."],
             'separator': ['-', 'Word separator. Defaults to "-".'],
-            "toc_height": [1,
-                           "Define the highest section level n (<hn>..<h6>) to"
-                           "include in the TOC. Section levels above are omitted."],
             "toc_depth": [6,
-                          "Define up to which section level n (<h1>..<hn>) to "
-                          "include in the TOC"]
+                          "Define the range of section levels to include in"
+                          "the Table of Contents. A single integer (b) defines"
+                          "the bottom section level (<h1>..<hb>) only."
+                          "Two digits seperated by a hyphen in between"
+                          " (```2-5```), define the top (t) and the bottom (b)"
```

Comment:
See my previous comment about including quotes to indicate the range should be a string.